src/openforms/submissions/api/serializers.py

```diff
@@ -477,3 +478,9 @@ class SubmissionStepSummarySerialzier(serializers.Serializer):
         required=True,
     )
     data = SubmissionComponentSummarySerializer(many=True)
+
+
+class SubmissionUpdateSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Submission
+        fields = ("privacy_policy_accepted",)
```

Comment:
I had expected this to be a field for the submission completion serializer and not a separate endpoint with additional call overhead?

Reply:
Ah I wasn't sure, since the `_complete` endpoint doesn't receive data in the request body so I thought it would be weird if it were to receive the `privacy_policy_accepted` field. Also it needs to be persisted to the database, so I thought it'd make more sense to have it in another endpoint than have the `_complete` endpoint update the submission. But indeed there is overhead for an extra call :shrug:

Reply:
it's a POST call already, so it makes perfect sense to add any "finalizers" in the request body.

The serializer can easily be adapted to persist this on the submission instance too, so let's do it like that.